.changeset/mighty-dots-boil.md

```
@@ -0,0 +1,17 @@
+---
+"@belgattitude/eslint-config-bases": minor
+---
+
+All plugins to latest
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)